training/create_model.py

```diff
@@ -0,0 +1,54 @@
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.linear_model import RidgeCV
+import pandas.core.frame
+import pandas.core.series
+from sklearn.model_selection import RandomizedSearchCV
+import numpy as np
+
+
+def create_model(
+    classifier_name: str,
+    X_train: pandas.core.frame.DataFrame,
+    y_train: pandas.core.series.Series,
+):
+    """
+    Creates chosen model, performs tuning and fit
+    :param X_train: train set with features to use during fitting
+    :param y_train: train set with values to predict
+    :param classifier_name: name of a chosen classifier:
+            train_linear_regression or train_random_forest
+    :return: trained classifier of a chosen type
+    """
+    if "book" in X_train.columns:
+        print("You have to drop column 'book' first")
+        # X_train = X_train.drop(columns=["book"]) - alternatywa
+        return None
```

Comment:
Ja bym tu zrobił `raise ValueError()` - dostaliśmy błędne dane, to errorujemy.

Ogólnie - w Pythonie wyjątki i ich łapanie jest szybkie (w porównaniu do innych języków), i zalecany styl programowania to EAFP (it’s easier to ask for forgiveness than permission), a nie LBYL (look before you leap). Polecam np. https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/. Errory rzuca się często i gęsto generalnie, bo one od tego są.

Jak coś jest mocno nie tak, to generalnie lepiej jest rzucić wyjątek, niż zwracać coś arbitralnego, bo to użytkownik zepsuł i on powinien naprawić (np. podać inne dane).